core/validation/utils.py

```diff
@@ -0,0 +1,14 @@
+import pandas as pd
+
+
+def remove_duplicate_indexes(df: pd.DataFrame):
+    """Removes duplicate indexes, except the first instance.
+
+    Due to the pd.melt during transformation that maps a single spreadsheet row to multiple df rows, here we just keep
+    the first of each unique index (this refers to the spreadsheet row number). This ensures we only produce one error
+    for a single incorrect row in the spreadsheet.
+
+    :param df: a DataFrame
+    :return: the DataFrame without duplicate indexes
+    """
+    return df[~df.index.duplicated(keep="first")]
```

Comment:
Should we add a test for this? I suppose it is only one line, but might be useful

Reply:
I didn't as it was a one liner but I'll add one just to make sure it's functionality isn't altered unintentionally

Reply:
added